### PR TITLE
chore: update ko, es, fr locales

### PR DESCRIPTION
**Pull request by LPX55**
## Description

Also worth mentioning that the en locale file has been moved to match the structure of the other files.

## Notice

- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

### CODE REVIEW

**Comment by 0xdef1cafe** [comment]
@LPX55 can we please split the translation updates and the script into two separate PRs?

**Comment by LPX55** [comment]
Sure, let's do that.

File: src/assets/translations/ko/main.json [hunk 1]
```diff
@@ -763,25 +859,25 @@
...
+      "timestamp": "Timestamp",
+      "status": "Status",
+      "minerFee": "Miner fee",
+      "minerFeeCurrency": "Miner fee Currency",
+      "inputAmount": "Input amount",
+      "inputCurrency": "Input currency",
+      "inputAddress": "Input address",
+      "outputAmount": "Output amount",
+      "outputCurrency": "Output currency",
+      "outputAddress": "Output address"
```
Comment (0xdef1cafe) [inline src/assets/translations/ko/main.json:872]: i mean i don't speak korean but this looks like english to me

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
...ogTTQyLjk3ODA2NzQsNDIuNzcwODM4NSBDMzkuNDk1NDYwNiw0NC45MzY4Mzk3IDM2LjI0MDk5MjYsNDcuNDczMTU3MiAzMy4yOTI0MTY2LDUwLjMxOTcwMjEgTDI5LjcxNjY5MjEsNDYuNDI0NjI2MyBMNDIuOTc4MDY3NCw0Mi43NzA4Mzg1IFogTTU1LjczNDI3ODQsMC4wNjI4ODM5MDY2IEwzNi40MTk3Nzg4LDcuNzQ2MzY1NjggTDE4LjQxNzc3NSw3Ljc0NjM2NTY4IEw5Ljk0NzU5ODNlLTE0LC04LjE3MTI0MTQ2ZS0xNCBMNC4xODM3ODM5NiwyNS4yOTU2MzM3IEwwLjE2NjIxNTMyMSwzOC45ODgwMjIxIEwxMC42NDgwMjM1LDQ1LjI1NzMxNDcgQzE1LjYxMDczODEsNDguMjI1OTAwOSAyMC4wNTYxODMxLDUxLjk0MDI0MzcgMjMuODYwNTExOSw1Ni4yOTY0NjgxIEwyNy45NDE4NjYzLDYwLjk2OTkwNjggTDMyLjIyNTI4NjMsNTYuMDU1MTgwMiBDMzUuOTAwNjQ2OSw1MS44Mzc3NjYyIDQwLjE4MDgwNzgsNDguMjE3NTE2NCA0NC45NDY1NzgyLDQ1LjI5NDU3OTIgTDU1LjIyNTg1NTEsMzguOTg4MDIyMSBMNTEuNTIzOTU1OSwyNS4zMTQyNjYgTDU1LjczNDI3ODQsMC4wNjI4ODM5MDY2IEw1NS43MzQyNzg0LDAuMDYyODgzOTA2NiBaIiBpZD0iRmlsbC0xNiI+PC9wYXRoPgogICAgICAgICAgICAgICAgICAgIDwvZz4KICAgICAgICAgICAgICAgIDwvZz4KICAgICAgICAgICAgPC9nPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+)](https://shapeshift.com) [![ShapeShift](https://img.shields.io/badge/License-MIT-brightgreen)](https://github.com/shapeshift/web/blob/develop/LICENSE.md) [![ShapeShift](https://img.shields.io/badge/PRs-welcome-brightgreen)](https://github.com/shapeshift/web/pulls)
```
Comment (0xdef1cafe) [inline README.md:6]: why remove this badge?

File: src/assets/translations/ko/main.json (hunk 1 above)
Comment (LPX55) [reply src/assets/translations/ko/main.json:872]: It was intentional. The current CSV plugin isn't parsing unicode properly; seeing pretty much all non-Latin based character systems with broken text. It was something @XrosFade actually found, I meant to bring it up eventually but figured you guys had bigger tasks at hand ;) Should've mentioned it in the request, my bad!

File: README.md (hunk 2 above)
Comment (LPX55) [reply README.md:6]: lack of sleep 😅 fixed the readme to only remove the locale button

**Review by 0xdef1cafe — APPROVED** [review]

**Review by 0xApotheosis — APPROVED** [review]

**Comment by 0xApotheosis** [comment]
@LPX55 you'll need to pull the latest `develop` branch and merge it into this one (there should be no conflicts), then push, so we can merge it in.

**Review by cjthompson — APPROVED** [review]

**Pull request merged**